features/facebookImport/src/locales/en/explore.json

```diff
@@ -2,7 +2,7 @@
     "messages.title": "Your Messages",
     "messages.summary": "Your Facebook data contains {{messages}} messages from {{threads}} chats with {{people}} people.",
     "report.success": "Your report was sent to us. Thank you!",
-    "report.error": "Your report could not be sent. Please try again!",
+    "report.error": "Report could not be sent. Try again!",
```

Comment:
No "Please"?

Reply:
Arne told me to change it to that, it needed to be shorter so the text fits in one line. I can double check with "Please" to see if it fits, but I don't think so.

Reply:
Ah, OK. I liked the please. It seemed a bit more polite.